### PR TITLE
update login and logout button

### LINKED ISSUE #1009: [FEATURE] Update the style of Login and Logout text
**Is your feature request related to a problem? Please describe.**
Currently, the login and logout text are hyperlinks. We should change them to buttons.

**Describe the solution you'd like**
Change the login and logout hyperlink to a button

### PR DESCRIPTION
**Pull request by clav1d**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #1009**
Also updated the boldness of the Create Event button.

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Change the GUI of text input field box. [#991](https://github.com/rokwire/events-manager/issues/991)
 - Update the style of pagination in user event listing page. [#989](https://github.com/rokwire/events-manager/issues/989)
+- Update Login and Logout button. [#1009](https://github.com/rokwire/events-manager/issues/1009)
```
Comment (sandeep-ps) [inline CHANGELOG.md:15]: Please add the new entry to the end of the section as we discussed last week. Thanks.

File: static/main.css [hunk 2]
```diff
@@ -567,4 +568,17 @@ h5 {
...
 .pagination {
   margin-top: 22px;
+}
+
+.login-logout-button {
+  display: inline-block;
+  width: 123px;
+  height: 32px;
+  margin: 19px 0 13.2px 24px;
+  padding: 5px 39px 9px;
```
Comment (sandeep-ps) [inline static/main.css:578]: The text needs to be center aligned. Thanks.

**Comment by clav1d** [comment]
> I have left a couple of comments. Other changes look good to me. Thanks.

These problems are fixed in [a5035e3](https://github.com/rokwire/events-manager/pull/1012/commits/a5035e3e3e24d0c5c80e20a0d0846ba1a9a17f67)

**Comment by sandeep-ps** [comment]
Hi, @minump, Any quick feedback on this PR? If there are no issues, I would like to merge this soon. Thanks.

**Comment by minump** [comment]
> #1009

Thanks for the reminder.
In zeplin, I do not see the orange hover color for the login/logout buttons. The hover color is mentioned only for the mobile version. I maybe wrong here.
Also, the "Create event" button and "login"/"logout" buttons have different height. I think the "Create event" button should be changed to have lesser height (to match the login button). Maybe this can be tackled as a separate issue.

Other than these, the changes looks good to me.

**Comment by sandeep-ps** [comment]
> > #1009
> 
> Thanks for the reminder. In zeplin, I do not see the orange hover color for the login/logout buttons. The hover color is mentioned only for the mobile version. I maybe wrong here. Also, the "Create event" button and "login"/"logout" buttons have different height. I think the "Create event" button should be changed to have lesser height (to match the login button). Maybe this can be tackled as a separate issue.
> 
> Other than these, the changes looks good to me.

Hi, @minump,

Hover colors are mentioned for buttons on the Events Manager Home Styles page. We think it applies to all buttons. There wouldn't be a hover color for mobile screens as those work using touch. I agree that the create event button height needs to be updated based on the design. I will create another issue for that. Thanks.

**Comment by minump** [comment]
> > > #1009
> > 
> > 
> > Thanks for the reminder. In zeplin, I do not see the orange hover color for the login/logout buttons. The hover color is mentioned only for the mobile version. I maybe wrong here. Also, the "Create event" button and "login"/"logout" buttons have different height. I think the "Create event" button should be changed to have lesser height (to match the login button). Maybe this can be tackled as a separate issue.
> > Other than these, the changes looks good to me.
> 
> Hi, @minump,
> 
> Hover colors are mentioned for buttons on the Events Manager Home Styles page. We think it applies to all buttons with a white background. There wouldn't be a hover color for mobile screens as those work using touch. I agree that the create event button height needs to be updated based on the design. I will create another issue for that. Thanks.

Ooopss.. my bad.. hover for mobile screens... guess my brain just stopped working !!
I'll need to update catalog as well to have the hover. Thanks for pointing that out.
Approving the PR.

**Review by minump — APPROVED** [review]
Tested, looks fine. Approved.

**Comment by sandeep-ps** [comment]
Thanks, Minu. No worries! Sometimes, I feel like switching concepts between different screen sizes is not intuitive enough. 

I have created the new issue here: https://github.com/rokwire/events-manager/issues/1021

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**